DQMServices/Core/interface/DQMEDHarvester.h

```diff
@@ -24,13 +26,14 @@ class DQMEDHarvester
   // implicit assignment operator
   // implicit destructor
   virtual void beginRun(edm::Run const&, edm::EventSetup const&) {};
-  virtual void analyze(edm::Event const&, edm::EventSetup const&) final {};
+  virtual void produce(edm::Event&, edm::EventSetup const&) override final {};
```

Comment:
@gartung since you add `override` here and not for the other methods that are overriding, you are introducing a clang warning here:
```
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_2_X_2017-05-15-1100/src/DQMServices/Core/interface/DQMEDHarvester.h:28:16: warning: 'beginRun' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  virtual void beginRun(edm::Run const&, edm::EventSetup const&) {};
               ^
```